pipeline-notebooks/pipeline-general.ipynb

```diff
@@ -608,32 +609,33 @@
     "    return split_pdfs\n",
     "\n",
     "\n",
-    "def partition_file_via_api(file_tuple, **kwargs):\n",
+    "def partition_file_via_api(file_tuple, request, filename, content_type, **request_kwargs):\n",
```

Comment:
i am not sure if `request_kwargs` is the same thing as `request_data_kwargs` i put in https://github.com/Unstructured-IO/unstructured/pull/716/files#diff-1904aaa4e2be0c8efe726dbc2614b4080a6074d816d249cfbaddc1e1ab8738a0R22, if it is i think its better if we can sync them? I can change mine back to `request_kwargs`

Reply:
Hmm, yeah, would make sense to have them consistent. I think request_args is a bit more succint.